assets/login.html

```diff
@@ -25,7 +25,7 @@
             </div>
             @@END_IF@@
 
-            <input class="login-button" type="submit" value="Log in!">
+            <input class="generic-button thick-button" type="submit" value="Log in!">
```

Comment:
Please change generic-button to hellish-button to keep it namespaced like the other things in `shared.css`. Probably the same thing with thick-button and thin-button.